configs/ruby/GPU_VIPER.py

```diff
@@ -688,7 +688,7 @@ def construct_gpudirs(options, system, ruby_system, network):
         dir_cntrl.addr_ranges = dram_intf.range
 
         # Append
-        exec("system.ruby.gpu_dir_cntrl%d = dir_cntrl" % i)
+        exec("ruby_system.gpu_dir_cntrl%d = dir_cntrl" % i)
```

Comment:
Really, we should try to remove all `exec` calls, but this is OK for now.

Reply:
Agreed